src/containers/views/DataFileEdit.js

```diff
@@ -95,11 +118,16 @@ export class DataFileEdit extends Component {
         className="single">
         {errors.length > 0 && <Errors errors={errors} />}
         <div className="content-header">
-          <Breadcrumbs splat={filename} type="datafiles" />
+          <Breadcrumbs splat={directory || ""} type="data files" />
         </div>
 
         <div className="content-wrapper">
           <div className="content-body">
+            <InputPath
+              onChange={onDataFileChanged}
+              type="data files"
+              path={path} // TODO: Support using `relative_path` from API instead
```

Comment:
On `master`, `path == relative_path` for this resource.
In this PR, they are redefined.
So, do we parse it like below ?

``` javascript
const data_dir = path - relative_path
```